### ISSUE TITLE
fix 

### ISSUE THREAD

**Issue by foen** [Mimoja/ESP-NintendoSwitch#1]
the esp8266 not have more memory. you could write it to SD and then send it back from the SD

**Comment by Mimoja** [comment]
Yes, you are right. But it got the exploit to work, so i had no real need to

**Comment by Mimoja** [comment]
Will look into it tho, when i get another 8266 board
